### DISCUSSION TITLE
How to make Author's Twitter/X account reflect into meta tags ?

### DISCUSSION THREAD

**Discussion by iamwatchdogs** [cotes2020/jekyll-theme-chirpy#1967] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I have started using this project as a blog site and it was generated from chirpy-starter template but later I have added a few files from this very repo _(i.e., jekyll-theme-chirpy)_. And when I checked through the source content of the html generated, the author's twitter info is missing in the head tags section. 

This part of the docs says that adding those details in `data/_authors.yml` will reflect the details into the `meta` tag for "twitter:creator" as the value for name attribute.

<img width="894" alt="image" src="https://github.com/user-attachments/assets/1895c010-5c14-4fd5-bc01-843c186ef0c2">

And it do work for this repo's docs, 

<img width="1641" alt="image" src="https://github.com/user-attachments/assets/d9bea951-923b-4893-bc9f-b149cf59a96a">

And even though I did the same things which was instructed by the docs, There's no such meta tag within my blog site...

<img width="1618" alt="image" src="https://github.com/user-attachments/assets/80b90656-4c5a-4982-af9d-457cd31175b6">

I did everything as mentioned within the docs, Is there something I'm missing ???...

Here's the repo of the blog site: <https://github.com/grow-with-open-source/notebook>

### Operations you have already tried

I did everything as mentioned within docs, I tried to trace out the file which generates the meta tags for twitter, but I failed... I don't know what else to do. Can anyone help me out with my situation ? Is there anything wrong with my repo ?

Here's the repo of the blog site: <https://github.com/grow-with-open-source/notebook>

**Comment by kungfux** [comment]
Please also check `_config.yml` to make sure `twitter` parameter is present.

```yml
twitter:
  username: twitter_username # change to your Twitter username
```

**Reply by iamwatchdogs** [reply]
I have configured that in `_config.yml`, here's the [permalink](https://github.com/Grow-with-Open-Source/Notebook/blob/0cfd740ceb4273e5dfcb814dbfea22b1ce0d0475/_config.yml#L35) for that config. And yet it's not working.

**Reply by iamwatchdogs** [reply, marked as answer]
I'm soo sorry, I just checked on the deployed build on GitHub pages. It does have the meta tag for `twitter:creator`. I was observing these changes though localhost and it somehow got added during the production build. I'm so sorry for creating this Q&A discussion.
